src/ngAnimate/animate.js

```diff
@@ -1869,7 +1869,7 @@ angular.module('ngAnimate', ['ng'])
           return;
         }
 
-        if (!staggerTime && styles) {
+        if (!staggerTime && styles && Object.keys(styles).length > 0) {
```

Comment:
Is it possible that styles could be a string? If so then calling `Object.keys` would throw an exception.